src/library/Uno.Themes/Helpers/TextBoxHeaderStateTrigger.cs

```diff
@@ -0,0 +1,67 @@
+using Microsoft.UI.Xaml;
```

Comment:
Until we finish https://github.com/unoplatform/Uno.Themes/issues/1447, we still need this to compile against the UWP APIs as well :/ 

so we'd need an `#if WinUI` here